platform/android/jni_utils.cpp

```diff
@@ -167,8 +167,14 @@ jvalret _variant_to_jvalue(JNIEnv *env, Variant::Type p_type, const Variant *p_a
 			v.obj = arr;
 
 		} break;
-
-			// TODO: This is missing 64 bits arrays, I have no idea how to do it in JNI.
+		case Variant::PACKED_FLOAT64_ARRAY: {
+			Vector<double> array = *p_arg;
+			jdoubleArray arr = env->NewDoubleArray(array.size());
+			const double *r = array.ptr();
+			env->SetDoubleArrayRegion(arr, 0, array.size(), r);
+			v.val.l = arr;
+			v.obj = arr;
+		} break;
```

Comment:
I think you also need to update the mappings in `get_jni_type(...)` and `get_jni_sig(...)`